entries/entry04.md

```diff
@@ -1,8 +1,28 @@
 # Entry 4
-##### X/X/XX
+##### 2/9/20
+## Recap
+  This past month In class, I have been searching for more templates, mostly animations and character templates so I can edit, add on and implement my own animations. Outside of class, I have been coding and tinkering in the Unity engine and also trying out Unity projects and also searched and watched any videos on how controller support works.
 
-Text
 
-[Previous](entry03.md) | [Next](entry05.md)
 
-[Home](../README.md)
\ No newline at end of file
+## Engineering Design Process
+In the Engineering Design Process, I thought I was close and ready to move onto step 5 which is **prototyping**, but I took a few steps back and stayed in the **planning process** which is step 4. I feel like I want to plan even more than I already did. I did not feel confident creating my prototype just yet. I will be reaching the prototyping stage soon but I just want to make sure that things go smoothly in the beginning and I can get more work done rather than thinking too much .
+
+
+## Knowledge
+In class I have been learning **Java** and it correlates to **C#** very well. Most things are very similar therefore it makes my working process simpler.
+
+## Skills
+A skill I have obtained and practiced more was **consideration**. Being open to other ideas and recommendations was beneficial to me and opened up more opportunities. Considering other ideas generally made me feel less cramped and made me feel like I have many different things to choose from. After all, this is a freedom project so consideration is a big factor. I can move freely, take in many suggestions and implement some into my own ideas to make an even better project. Another skill I have practiced was **embracing failure**. Being open to failing really helped me... I let it motivate me and allowed myself to work twice as harder. The frustration when searching for what IDE to put my code in for unity pushed me more and allowed me to google everywhere and tinker on different IDE programs. I finally figured that the C+ code goes in the Unity engine itself. Making mistakes and being clueless generally pushed me and I can say that tinkering definitely helps solve problems.
+
+
+## Future plans for this week and the following
+The next following weeks I plan to ask for my personal laptop to use in class because Unity does not run on the macs we use. Having the Unity engine during class time would help and be more efficient than the process I already am in.
+ 
+
+## Sources
```

Comment:
As I've discussed, don't just plop the sources on here. **Show** _how_ you're using them.